backend/routes/stripe.js

```diff
@@ -62,7 +62,13 @@ module.exports = function (router) {
     const shopConfig = getConfig(req.shop.config)
     const web3Pk = shopConfig.web3Pk || networkConfig.web3Pk
 
-    if (!web3Pk || !shopConfig.stripeBackend) {
+    const valid = await stripeUtils.webhookValidation(
```

Comment:
nit (not for this line but could not put a comment above): since we are touching this route, might as well document it a bit. add a short jsdoc header for this route explaining when it gets called and the argument expected in the body of the request.

Reply:
Sure, will do.